### ISSUE TITLE
Example notebooks for using TPOT on regression problems

### ISSUE THREAD

**Issue by rhiever** [rhiever/tpot#257]
@slcott expressed interest in making some TPOT example notebooks for using TPOT on regression problems. I'm 👍 on this idea. If you make any notebooks, for example with the Boston house price data set, please PR them directly to the `master` branch.

**Comment by slcott** [comment]
Sign me up. 

Let's see how well TPOT does.

**Comment by rhiever** [comment]
I've added a [very basic example](http://rhiever.github.io/tpot/examples/Boston_Example/) using the Boston data set to the docs. Feel free to expand upon it.

**Comment by slcott** [comment]
I played around with the notebook. Trying to come up with ways to analyze the dataset. I'm wondering about overfit from tpot. The exported regressor has an r2 score of 1.0 when run on the training set. It still performs better on the test set than a simple DecisionTreeRegressor. Was thinking about adding a validation dataset. Maybe this is is overkill for a tutorial. Thoughts?

[Boston Housing Prices.ipynb.zip](https://github.com/rhiever/tpot/files/460012/Boston.Housing.Prices.ipynb.zip)

**Comment by rhiever** [comment]
I think it's expected to achieve a training score of 1.0 if you run `fit` and `score` on the same training features and labels. Basically, you're training and testing on the same data there. If you run TPOT with `verbosity=2` (you can set this in the constructor when you create a new TPOT instance), then you will see the internal k-fold CV scores, which will probably be in the 0.8-0.9 range.

Another venue I would explore is running TPOT for a short time period (as you have), and then running it for a longer time period (say 8 hours?) using the `max_time_mins` parameter. Will it find an even better pipeline for the data set?

**Comment by rhiever** [comment]
Hi @slcott -- are you still working on these notebooks?
